cmd/stack.go

```diff
@@ -90,3 +115,28 @@ func setupStackCommand() *cobra.Command {
 		shellInitCommand)
 	return cmd
 }
+
+func availableServicesAsList() []string {
+	var available []string
```

Comment:
Nit: Not that it matters much here because we are only dealing with a handful of `availableServices` but in general, when the size of the slice can be determined beforehand you can preallocate the memory for it by doing:

```
available := make([]string, len(availableServices)
```

And then you don't even need the `append` because you can do something like:

```
i := 0
for aService := range availableServices {
  available[i] = aService
  i++
}
```

But again, this is just to show a _general_ pattern when converting the keys of a `map[string]interface{}` to a `[]string`. What you have here is totally fine since we are only dealing with a handful of keys.

Reply:
You're right, we can optimize it here :)

Fixed.